Source/WebGPU/WebGPU/Texture.h

```diff
@@ -162,6 +163,16 @@ class Texture : public WGPUTextureImpl, public RefCounted<Texture>, public CanMa
     mutable WeakHashSet<CommandEncoder> m_commandEncoders;
     id<MTLSharedEvent> m_sharedEvent { nil };
     uint64_t m_sharedEventSignalValue { 0 };
-};
+} SWIFT_SHARED_REFERENCE(retainTexture, releaseTexture);
 
 } // namespace WebGPU
+
+inline void retainTexture(WebGPU::Texture* obj)
+{
+    WTF::retainRefCounted(obj);
```

Comment:
...and we can call this 'ref'. A free function that does `obj->ref()` is `ref(obj)`. Changing the argument positions doesn't need to change the name.